salvo/src/lib/benchmark/fully_dockerized_benchmark.py

```diff
@@ -0,0 +1,37 @@
+"""
+This module contains the methods to perform a nighthawk benchmark using
+containers for the scripts, nighthawk binaries, and envoy
+
+https://github.com/envoyproxy/nighthawk/blob/master/benchmarks/README.md
+"""
+
+import logging
+
+from src.lib.benchmark.base_benchmark import BaseBenchmark
+
+log = logging.getLogger(__name__)
+
+
+class Benchmark(BaseBenchmark):
+
+  def __init__(self, **kwargs):
```

Comment:
What arguments are we expecting to be passing here and can we name them instead of using `kwargs`? Listing the arguments specifically would help us to think about code structure and would make the code more readable.

Reply:
Sure.  For now it's just one argument.  I will update.   There are other places where `kwargs` is used like this so I'll take a look at those also.